src/installer/corehost/cli/apphost/static/CMakeLists.txt

```diff
@@ -78,62 +79,131 @@ if(CLR_CMAKE_TARGET_WIN32)
         bcrypt.lib
         RuntimeObject.lib
     )
-else()
+elseif(CLR_CMAKE_TARGET_LINUX)
     set(CORECLR_LIBRARIES
         ${CORECLR_STATIC_LIB_LOCATION}/libcoreclr_static.a
         ${CORECLR_STATIC_LIB_LOCATION}/libcoreclrpal.a
         ${CORECLR_STATIC_LIB_LOCATION}/libpalrt.a
         ${CORECLR_STATIC_LIB_LOCATION}/libeventprovider.a
         ${CORECLR_STATIC_LIB_LOCATION}/libnativeresourcestring.a
     )
-endif(CLR_CMAKE_TARGET_WIN32)
 
-if(CLR_CMAKE_TARGET_OSX)
-  find_library(COREFOUNDATION CoreFoundation)
-  find_library(CORESERVICES CoreServices)
-  find_library(SECURITY Security)
-  find_library(SYSTEM System)
-
-  LIST(APPEND CORECLR_LIBRARIES
-    ${COREFOUNDATION}
-    ${CORESERVICES}
-    ${SECURITY}
-    ${SYSTEM}
-  )
-endif(CLR_CMAKE_TARGET_OSX)
-
-# On OSX and *BSD, we use the libunwind that's part of the OS
-if(CLR_CMAKE_TARGET_FREEBSD)
-    find_unwind_libs(UNWIND_LIBS)
-
-    LIST(APPEND CORECLR_LIBRARIES
-      ${UNWIND_LIBS}
-    )
-endif(CLR_CMAKE_TARGET_FREEBSD)
+    # currently linking coreclr into the singlefilehost is only supported on linux
+    # the following code here would be needed if/when BSD and OSX are supported too
+    if(CLR_CMAKE_TARGET_OSX)
```

Comment:
Is this supposed to be in the `CLR_CMAKE_TARGET_LINUX` branch? Can both `CLR_CMAKE_TARGET_LINUX` and `CLR_CMAKE_TARGET_OSX` be true?

Reply:
The comment seems to imply that isn't true. If so, please comment it out with a argument/reason for why it would be uncommented.

Reply:
Just to make sure - are you suggesting to comment out the code that is not supposed to run?  (i.e. OSX etc..)

Reply:
Yes.